redisinsight/api/config/default.ts

```diff
@@ -27,7 +27,11 @@ const contentUri = posix.join('/', proxyPath, 'static', 'content');
 const defaultPluginsUri = posix.join('/', proxyPath, 'static', 'plugins');
 const pluginsAssetsUri = posix.join('/', proxyPath, 'static', 'resources', 'plugins');
 
-const socketPath = posix.join('/', proxyPath, 'socket.io');
+const socketProxyPath = trim(process.env.RI_SOCKET_PROXY_PATH || 'api', '/');
+
+const socketPath = posix.join('/', socketProxyPath, 'socket.io');
+
+// const socketPath = posix.join('/', proxyPath, 'socket.io');
```

Comment:
No point in leaving the old implementation in a comment. We can always recover it from git